src/constants/enum/contact/contact.ts

```diff
@@ -0,0 +1,4 @@
+export enum TypeContact {
+  PHONE = 'phone',
```

Comment:
Oriana habia quitado telefono? o estoy equivocado?
```suggestion
  WEBSITE = 'website',
```